src/paths/properties@percentiles@by-similarity.yaml

```
@@ -29,6 +29,12 @@ post:
         schema:
           type: object
           properties:
+            size:
+              type: integer
+              minimum: 0
+              maximum: 100
+              default: 10
+              description: Nombre de bien sur lequel faire l'analyse
```

:magic_wand: **chore**: typo
```suggestion
              description: Nombre de biens sur lequel faire l'analyse
```